### PR TITLE
Cleaning lady.. 

### PR DESCRIPTION
**Pull request by skedone**
- Added a README.md
- Added a composer.json configuration, so now you can just type ./bin/aerys (as the same as phpunit and other third party application)
- Updated the .gitignore
- Updated the dependencies

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 /vendor/
+/bin/
+/.idea
```
Comment (kelunik) [inline .gitignore:3]: `./idea` should be in your global `.gitignore`.

File: composer.json [hunk 2]
```diff
@@ -20,7 +20,10 @@
     "repositories": [
         {
             "type": "git",
-            "url": "git@github.com:rdlowrey/aerys"
+            "url": "git@github.com:amphp/aerys.git"
```
Comment (kelunik) [inline composer.json:23]: This is no longer required and already fixed.

File: composer.json [hunk 3]
```diff
@@ -20,7 +20,10 @@
...
         }
-    ]
+    ],
+    "config": {
+      "bin-dir": "bin"
+    }
```
Comment (kelunik) [inline composer.json:28]: Do we really need this?

File: .gitignore (hunk 1 above)
Comment (skedone) [inline .gitignore:3]: Right.

File: composer.json (hunk 3 above)
Comment (skedone) [inline composer.json:28]: Mah, usually I see that in a lot of projects. The vendor expose 

``` json
    "bin": [
        "phpunit"
    ],
```

so you can save a copy inside your `bin` directory, nothing more :)

Comment (kelunik) [inline composer.json:28]: I don't see a real reason to have that copy actually, it just saves a single path segment.

Comment (skedone) [inline composer.json:28]: Okidokie :+1:  Closing the pull request :)

**Pull request closed without merging**